staging/src/k8s.io/kms/internal/plugins/base64/setup-local-registry.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Let's move the logic in this script to a function in `test/e2e/testing-manifests/auth/encrypt/setup-cluster.sh`